packages/dappmanager/test/modules/compose/editor/composeAliases.test.ts

```diff
@@ -0,0 +1,191 @@
+import "mocha";
+import { expect } from "chai";
+import { ComposeFileEditor } from "../../../../src/modules/compose/editor";
+import params from "../../../../src/params";
+import { shellSafe } from "../../../testUtils";
+import fs from "fs";
+import { parseServiceNetworks } from "../../../../src/modules/compose/networks";
+
+describe("compose service editor", () => {
+  const exampleCompose = `
+version: '3.5'
+services:
+  goerli-geth.dnp.dappnode.eth:
+    container_name: DAppNodePackage-goerli-geth.dnp.dappnode.eth
+    dns: 172.33.1.2
+    environment:
+      - 'EXTRA_OPTIONS=--http.api eth,net,web3,txpool'
+    image: 'goerli-geth.dnp.dappnode.eth:0.4.12'
+    logging:
+      driver: json-file
+      options:
+        max-size: 10m
+        max-file: '3'
+    networks:
+      dncore_network:
+        aliases:
+          - goerli-geth.dappnode
+    ports:
+      - '30303'
+      - 30303/udp
+      - 30304/udp
+    restart: always
+    volumes:
+      - 'goerli:/goerli'
+    labels:
+      dappnode.dnp.dnpName: goerli-geth.dnp.dappnode.eth
+      dappnode.dnp.version: 0.4.12
+volumes:
+  goerli: {}
+networks:
+  dncore_network:
+    external: true
+`;
+
+  const dnpName = "example";
+  const serviceName = "goerli-geth.dnp.dappnode.eth";
+  const dnpRepoExamplePath = process.cwd() + "/dnp_repo/example";
```

Comment:
Don't create files in random directories that are not gitignored. You can use this utilities to create artifacts in predictable directories that are already gitignored
https://github.com/dappnode/DNP_DAPPMANAGER/blob/057b1eee877b9bc19c2fee9593cbbcd5de46a2e2/packages/dappmanager/test/testUtils.ts#L18
https://github.com/dappnode/DNP_DAPPMANAGER/blob/057b1eee877b9bc19c2fee9593cbbcd5de46a2e2/packages/dappmanager/test/testUtils.ts#L58

Reply:
this folder I create is necessary since the `ComposeEditor` looks at this path for composes files in the dappmanager container. I just create the folder `/example` inside the already existing `dnp_repo` folder created by other tests

Also this path ( `dnp_repo`) is already ignored by the gitignore https://github.com/dappnode/DNP_DAPPMANAGER/blob/057b1eee877b9bc19c2fee9593cbbcd5de46a2e2/.gitignore#L2

Reply:
I would use `testDir` for consistency with all other tests in the DAPPMANAGER

Reply:
Isn't that overcomplicating the test?
Editor get compose path at the end from https://github.com/dappnode/DNP_DAPPMANAGER/blob/057b1eee877b9bc19c2fee9593cbbcd5de46a2e2/packages/dappmanager/src/utils/getPath.ts#L83

I would have to overwrite the variable `REP_DIR` to `./test_files` just for this test?
https://github.com/dappnode/DNP_DAPPMANAGER/blob/057b1eee877b9bc19c2fee9593cbbcd5de46a2e2/packages/dappmanager/src/params.ts#L30